core/src/main/java/org/infinispan/notifications/cachemanagerlistener/CacheManagerNotifierImpl.java

```diff
@@ -133,7 +135,10 @@ private void invokeListener(ListenerInvocation listener, EventImpl e) {
       } catch (Exception x) {
          // Only cache entry-related listeners should be able to throw an exception to veto the operation.
          // Just log the exception thrown by the invoker, it should contain all the relevant information.
-         log.error(x);
```

Comment:
The reason the stack trace isn't logged is because it is using the error(Object) method which just invokes toString.  If you instead call log.error(String, Throwable) method it will print the stack trace.

Although I wonder if we want to only print the stack trace if tracing is enabled and leave it as is for when tracing is not?